README.md

```diff
@@ -1,3 +1,38 @@
-# Exploring the set of interoperable features in the Web Platform.
+# [`feature-set`](https://github.com/web-platform-dx/feature-set): exploring the set of interoperable features of the web platform
 
-See the [2022 backgrounder](2022-backgrounder.md) for details.
\ No newline at end of file
+By the W3C WebDX Community Group and contributors
+
+`feature-set` is an effort to build a shared catalog of features of the web platform.
+By creating a common list of features and their definitions, `feature-set` aims to improve understanding of what web developers get and want from the web.
+`feature-set` does this by:
+
+* Creating feature definitions, which identify and describe capabilities of the web
+* Generating _Baseline_ support data, which summarizes the availability of web features across key browsers and releases
+* Publishing the `web-features` npm package, which bundles feature identifiers with Baseline statuses
+
+From `Array` to Cascade Layers to Offscreen Canvas, `feature-set` identifies, defines, and categorizes capabilities of the web platform that web developers care about.
+By creating a common nomenclature for web platform features, `feature-set` intends to support communication between web developers, browser vendors, standardization bodies, and other ecosystem participants.
+
+## Get started
+
+To get feature identifiers, Baseline support data, or cross-references to other resources (such as specifications and caniuse), use [the `web-features` npm package](https://www.npmjs.com/package/web-features).
+[Read the package README](./packages/web-features/README.md) for more information.
+
+To contribute to feature identification, definition, Baseline support statuses, or to update the status on MDN, jump to [Contribute](#contribute).
+
+## Background
+
+To learn more about why this project was started and some of the problems its responding to, read the [2022 backgrounder](./2022-backgrounder.md).
+
+## Contribute
+
+The project is in its early stages, so we’re still developing tools and processes to help improve and expand our work.
+To get started contributing, take a look the following:
+
+* Learn more about [the WebDX Community Group](https://github.com/web-platform-dx/admin/blob/main/charter.md)
+* Go to [open issues](https://github.com/web-platform-dx/feature-set/issues) to propose new feature groups or report a bug
+* Go to [open PRs](https://github.com/web-platform-dx/feature-set/pulls) to review incoming feature groups and updates
+
+## License
+
+You’re free to TODO: license file needed.
```

Comment:
We ought to have something like this here _and_ a LICENSE file for the whole repo. See also: https://github.com/web-platform-dx/feature-set/pull/163

```suggestion
You are free to use, copy, modify, and distribute `feature-set`, as long as you include the required notices. Read the LICENSE file for details.
```